content/pages/about.rst

```diff
@@ -76,20 +76,21 @@ Directions
 6. Turn right onto SW 26th.
 7. Turn right onto SW Campus Way.
 
-Arrive at Kelley Engineering Center! Hope you enjoy your visit. 
+Arrive at Kelley Engineering Center! Hope you enjoy your visit.
 
 
 Parking
 ~~~~~~~
 
-Parking may be a little bit tricky this year, as the parking lot 
+Parking may be a little bit tricky this year, as the parking lot
 just north of Kelley is closed, and there is a footbal scrimmage
-on the same day as Barcamp.  We recommend you park in the 
+on the same day as Barcamp.  We recommend you park in the
 highlighted parking lots close to Kelley on the map below! Parking
 is free everywhere, so if you find a space go ahead and take it.
 
-.. figure:: ../theme/img/parking2.png
+.. image:: /images/BBCParkingMap.png
```

Comment:
I think we've resolved the issue. It took a little digging but it's working now on the Jenkins site. If you see any issues though let me know.